src/bin/cargo-miri.rs

```diff
@@ -84,6 +84,31 @@ fn get_arg_flag_value(name: &str) -> Option<String> {
     }
 }
 
+/// Returns the path to the `miri` binary
+fn find_miri() -> PathBuf {
+    let mut path = std::env::current_exe().expect("current executable path invalid");
+    path.set_file_name("miri");
+    path
+}
+
+fn cargo() -> Command {
+    if let Ok(val) = std::env::var("CARGO") {
```

Comment:
I would like to use `env;:var_os` when ever it is possible.

Reply:
That probably makes sense, if we don't need it to really be UTF-8... do you want to submit a PR?